apps/kaizen/models.py

```
@@ -1,3 +1,56 @@
+        return self.title
+
+
+# SubTask model
+class SubTask(BaseModel):
+    task = models.ForeignKey(Task, on_delete=models.CASCADE, related_name="subtasks")
+    title = models.CharField(max_length=255)
+    completed = models.BooleanField(default=False)
+    completed_time = models.DateTimeField(blank=True, null=True)
+    is_all_day = models.BooleanField(default=False)
+    start_date = models.DateTimeField()
+    time_zone = models.CharField(max_length=50, blank=True, null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)